mkdocs.yml

```diff
@@ -98,37 +98,12 @@ plugins:
   - minify:
       minify_html: true
   # - social # @insiders
-  - mkdocstrings:
-      default_handler: python
-      handlers:
-        python:
-          rendering:
-            show_source: true
   - mkdocs-jupyter:
       include_source: true
       ignore_h1_titles: true
       execute: true
   - tooltips
-  - markmap:
-      base_path: docs
-      encoding: utf-8
-      file_extension: .mm.md
-      d3_version: 6.7.0
-      lib_version: 0.11.5
-      view_version: 0.2.6
-  - kroki:
-      ServerURL: https://kroki.io
-      EnableBlockDiag: true
-      Enablebpmn: true
-      EnableExcalidraw: true
-      EnableMermaid: true
-      DownloadImages: false
-      DownloadDir: docs/assets/kroki_generated
```

Comment:
:+1: I would have dropped `kroki` as well. But `markmap` gives you the ability of adding mindmaps with markdown for better explanation of steps.

I would suggest you keep it for now. Why? Well, you need sequential steps to produce a certain visualization. You can think of each intermediate output as a node on a mindmap, followed by an operation. A mindmap could be helpful here showing this.

- https://github.com/neatc0der/mkdocs-markmap
- https://markmap.js.org/
- example: https://markmap.js.org/repl

![image](https://user-images.githubusercontent.com/10201242/172761235-7f989dce-f220-42e2-bc1c-0ea26d076771.png)

Reply:
For `kroki` you get the following:

![kroki_cheatsheet_20210515_v1 1_EN-1](https://user-images.githubusercontent.com/10201242/172764757-3115cd30-8e91-4209-816e-9ff6be90b3c4.png)

Reply:
We can add these back if we find a compelling use case.